docs/README.md

```diff
@@ -22,7 +22,26 @@ The Media app allows you to display image and video assets using a single block.
  }
 ```
 
-2. In any desired theme template, add the `media` block with the desirable props. For example:
+Now, you are able to use all blocks exported by the `store-media` app. Check out the full list below:
+
+| Block name                | Description                                                                                                                                                                                                                                               |
+| ------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `media`                   | This block is capable to display a single image or video.                                                                                                                                                                                                 |
+| `list-context.media-list` | This block allows you to display a list of images and videos in your store with a higher degree of composability, since you can use it along with other `list-context` or `slider-layout` blocks to create a more flexible and customizable image slider. |
```

Comment:
```suggestion
| `list-context.media-list` | This block allows you to display a list of images and videos in your store with a higher degree of composability. You can use it along with other `list-context` or `slider-layout` blocks to create different sliders and layouts. |
```

I would remove the `[...] and customizable image slider` part, since the best part of using this is that you could create sliders (or just plain lists) of anything (images, videos, products...).

Reply:
You're right! I copied it from the docs on store-image and forgot to change that part hahah